.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./node_modules/gts/",
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "off"
```

Comment:
I think we'd want this on

Reply:
I agree but at the same time I was unsure what to put instead of `any` for the generic types of `d3.Selection<SVGGElement, any, any, any>`. I initially suppressed warning for that specific line but realized it will be used quite frequently.

Reply:
Should I create a type alias like below and re-enable the rule maybe?
```ts
// eslint-disable-next-line @typescript-eslint/no-explicit-any
type D3Selection<
  GElement extends d3.BaseType,
  Datum = any,
  PElement extends d3.BaseType = any,
  PDatum = any
> = d3.Selection<GElement, Datum, PElement, PDatum>;
```

Reply:
We use something pretty similar to this internally for our d3 charts.  But instead of `any`, we use `unknown`.  In general, whenever `any` seems like a good idea, we should use `unknown` instead, unless an external library makes it impossible.

Reply:
Good call, I'll try using `unknown` next time.